CRM/NcnCiviZoom/Form/Settings.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use CRM_NcnCiviZoom_ExtensionUtil as E;
+
+/**
+ * Form controller class
+ *
+ * @see https://wiki.civicrm.org/confluence/display/CRMDOC/QuickForm+Reference
+ */
+class CRM_NcnCiviZoom_Form_Settings extends CRM_Core_Form {
+  public function buildQuickForm() {
+
+    // add form elements
+    $this->add('password', 'api_key', ts('Api Key'), array(
+      'size' => 48,
+    ), TRUE);
+    $this->add('password', 'secret_key', ts('Secret Key'), array(
+      'size' => 48,
+    ), TRUE);
+    $this->add('text', 'base_url', ts('Base Url'), array(
+      'size' => 48,
+    ), TRUE);
+    $this->add('text', 'custom_field_id', ts('Custom Field Id'), array(
```

Comment:
some help text will be really helpful for the user to enter the custom_field_id in an expected format.